src/lib.rs

```diff
@@ -1555,19 +1493,10 @@ mod tests {
         }
 
         let alpn = [Buffer::from("h3")];
-        let res = Configuration::new(
-            &registration,
-            &alpn,
-            #[cfg(feature = "preview-api")]
-            Settings::new()
-                .set_peer_bidi_stream_count(100)
-                .set_peer_unidi_stream_count(3)
-                .set_stream_multi_receive_enabled(true),
-            #[cfg(not(feature = "preview-api"))]
-            Settings::new()
-                .set_peer_bidi_stream_count(100)
-                .set_peer_unidi_stream_count(3),
-        );
+        let settings = Settings::new()
```

Comment:
What about the use of `#[cfg(not(feature = "preview-api"))]` that was here before?

Reply:
We do not have preview apis in generated bindings.
We need to support generating preview apis for bindings first to support the safe wrappers. Let me know if rust support for preview apis are required.